src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.profiles.active=@activatedProperties@
```

Comment:
In our Procfile, we say java -jar -Dspring.profiles.active=prod ourAppName.war.
'prod' value will be placed here when deploying to heroku.
'dev' will be placed here when running on your machine (since you don't pass it anything, it takes the default specified in the pom which is 'dev').
Notice the relation between this and our new application.properties files... application-dev.properties and application-prod.properties